### PR TITLE
[inner-1953] adjust the txid logic for the XA scenario

### PR DESCRIPTION
**Pull request by wenyh1**
Reason:  
  BUG inner 1953
Type:  
  BUG/Improve  
Influences：  
   fix xx

### CODE REVIEW

File: src/main/java/com/actiontech/dble/backend/mysql/nio/handler/MultiNodeLoadDataHandler.java [hunk 1]
```diff
@@ -712,10 +712,10 @@ void handleEndPacket(MySQLPacket curPacket, AutoTxOperation txOperation, boolean
             if (txOperation == AutoTxOperation.COMMIT) {
                 session.checkBackupStatus();
                 session.setBeginCommitTime();
-                handler.commit();
+                handler.commit(null);
```
Comment (PanternBao) [inline src/main/java/com/actiontech/dble/backend/mysql/nio/handler/MultiNodeLoadDataHandler.java:715]: add a new interface method for impcilit commit

File: src/main/java/com/actiontech/dble/backend/mysql/nio/handler/transaction/xa/stage/XARollbackStage.java [hunk 2]
```diff
@@ -26,9 +26,13 @@
 public class XARollbackStage extends XAStage {
 
     private static Logger logger = LoggerFactory.getLogger(XARollbackStage.class);
-    private boolean lastStageIsXAEnd;
+    protected boolean lastStageIsXAEnd = true;
```
Comment (PanternBao) [inline src/main/java/com/actiontech/dble/backend/mysql/nio/handler/transaction/xa/stage/XARollbackStage.java:29]: why?

Comment (wenyh1) [reply src/main/java/com/actiontech/dble/backend/mysql/nio/handler/transaction/xa/stage/XARollbackStage.java:29]: given default values

File: src/main/java/com/actiontech/dble/backend/mysql/nio/handler/transaction/xa/stage/XACommitStage.java [hunk 3]
```diff
@@ -88,12 +88,6 @@ public void onConnectionClose(MySQLResponseService service) {
         XAStateLog.saveXARecoveryLog(session.getSessionXaID(), service);
     }
 
-    @Override
```
[the remark this answers is not in the document]
Comment (wenyh1) [reply src/main/java/com/actiontech/dble/backend/mysql/nio/handler/transaction/xa/stage/XACommitStage.java]: to consider the next pr

File: src/main/java/com/actiontech/dble/backend/mysql/nio/handler/MultiNodeLoadDataHandler.java (hunk 1 above)
Comment (wenyh1) [reply src/main/java/com/actiontech/dble/backend/mysql/nio/handler/MultiNodeLoadDataHandler.java]: ok

**Review by PanternBao — APPROVED** [review]
lgtm

**Pull request merged**